server/supernova/audio_backend/portaudio_backend.hpp

```diff
@@ -285,18 +293,25 @@ class portaudio_backend : public detail::audio_backend_base<sample_type, float,
             super::fetch_inputs(inputs, blocksize_, input_channels);
             engine_functor::run_tick();
             super::deliver_outputs(outputs, blocksize_, m_hwOutputChannels);
+            if (IsClipping)
```

Comment:
I moved clipping here from before this `while` loop, since it turned out that we are getting the `outputBuffer`, i.e. `out` populated after `super::deliver_outputs`.